modules/electrophysiology_browser/jsx/components/electrophysiology_session_panels.js

```diff
@@ -555,16 +560,85 @@ class FilePanel extends Component {
       </Panel>
     );
   }
+
+    /**
+     * Called by download all or download annotations button to
+     * update the annotation files before downloading
+     */
+    handleClick() {
+        console.log('The link was clicked.');
+        console.log(this.state.id);
+
+        fetch(this.props.url, {
+            method: 'PUT',
+            body: {physioFileID: this.state.id},
+        }).then((response) => {
+            if (!response.ok) {
+                console.error(response.status);
+                return;
+            }
+
+            response.json().then((data) => {
+                let msgType = 'success';
+                let message = 'Your files will be downloaded in 2 seconds!';
+                this.showAlertMessage(msgType, message);
+                console.log(message);
```

Comment:
Not needed?

Reply:
Do you have cases where an error can occur? For example, if the physiologicalID is not found.

Reply:
Yes there are cases where the file is not found or the file path is incorrect/not found as well